torch/export/__init__.py

```diff
@@ -182,6 +182,12 @@ def __post_init__(self) -> None:
             == assertion_dep_token_index
         )
 
+    def inputs(self) -> List[str]:
+        return list(self.inputs_to_parameters.keys()) + list(self.inputs_to_buffers.keys()) + self.user_inputs
+
+    def outputs(self) -> List[str]:
+        return list(self.buffers_to_mutate.keys()) + self.user_outputs
```

Comment:
Does this also need the backward outputs? (https://www.internalfb.com/code/fbsource/[9b6229be95c1]/fbcode/configerator/structs/aps/models/ir_graph.thrift?lines=301)